php/helpers.php

```diff
@@ -43,6 +43,11 @@ function block_field( $name, $echo = true ) {
 		$value = strval( $value );
 	}
 
+	// Return immediately for EditorsKit attributes
```

Comment:
No period at the end of this comment is causing Travis to fail the `phpcs` check. Seems minor, but could you please change it to:
`// Return immediately for EditorsKit attributes.`

Reply:
Hi @lukecarbis, I should commented on this PR, but the current approach we're considering is a different branch entirely, [add/allowed-filter](https://github.com/getblocklab/block-lab/pull/373#issuecomment-517110834)